### PR TITLE
Teams docs

### PR DESCRIPTION
**Pull request by merit-maita**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Adding a new method for configuring the bot in prod envs.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 1]
```diff
@@ -623,10 +623,13 @@ def validate_auth_header(headers: dict) -> bool:
         return False
 
     integration_context: dict = get_integration_context()
+    # demisto.debug(f"integration context is: {integration_context}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:626]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 2]
```diff
@@ -623,10 +623,13 @@ def validate_auth_header(headers: dict) -> bool:
...
     open_id_metadata: dict = json.loads(integration_context.get('open_id_metadata', '{}'))
+    # demisto.debug(f"open_id_metadata from integration context is: {open_id_metadata}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:628]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 3]
```diff
@@ -623,10 +623,13 @@ def validate_auth_header(headers: dict) -> bool:
...
     keys: list = open_id_metadata.get('keys', [])
 
     unverified_headers: dict = jwt.get_unverified_header(jwt_token)
+    # demisto.debug(f"unverified headers are: {unverified_headers}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:632]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 4]
```diff
@@ -672,7 +675,7 @@ def validate_auth_header(headers: dict) -> bool:
         # Didn't find requested key in new keys
         demisto.info('Authorization header validation - failed to find relevant key')
         return False
-
+    # demisto.debug(f"now key object is: {key_object}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:678]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 5]
```diff
@@ -661,7 +664,7 @@ def validate_auth_header(headers: dict) -> bool:
         # Didn't get new keys
         demisto.info('Authorization header validation - failed to get keys')
         return False
-
+    # demisto.debug(f"keys are: {keys}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:667]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 6]
```diff
@@ -635,7 +638,7 @@ def validate_auth_header(headers: dict) -> bool:
         if key.get('kid') == key_id:
             key_object = key
             break
-
+    # demisto.debug(f"first key_object is: {key_object}")
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:641]: Needed?

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md [hunk 7]
```diff
@@ -125,7 +125,24 @@ Before you can create an instance of the Microsoft Teams integration in Cortex X
 ### Create the Demisto Bot in Microsoft Teams
 
 
-#### Using the Developer Portal
+#### Creating the Demisto Bot for Production environment using Microsoft Azure Portal
```
Comment (tkatzir) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md:128]: Needs to be reviewed by Tech Writers

**Review by tkatzir — CHANGES_REQUESTED** [review]
Good. Need to remove debug code, and have the Tech Writers check the English text.

**Comment by ShirleyDenkberg** [comment]
@tkatzir Doc review completed.

**Review by tkatzir — APPROVED** [review]
Great!

**Pull request merged**